phpstan.neon.dist

```diff
@@ -30,7 +30,3 @@ parameters:
         -
             message: '#^Array has 2 duplicate keys with value .ClassWithConstants\.SOME_KEY. \(\\Doctrine\\Tests\\Annotations\\Fixtures\\ClassWithConstants::SOME_KEY, \\Doctrine\\Tests\\Annotations\\Fixtures\\ClassWithConstants::SOME_KEY\)\.$#'
             path: '%currentWorkingDirectory%/tests\Doctrine\Tests\Annotations\DocParserTest.php'
-        # bug in doctrine/lexer
-        -
-            message: '#^Call to static method PHPUnit\\Framework\\Assert::assertNull\(\) with array will always evaluate to false\.$#'
-            path: '%currentWorkingDirectory%/tests\Doctrine\Tests\Annotations\DocLexerTest.php'
```

Comment:
Does this related to one of the other 2 commits? If yes, please squash it with that commit, and explain the relationship in the commit message.

Reply:
Yes, the error message is no longer reported.  I noticed that PHPStan was throwing an error after my initial commit in [this Travis build](https://travis-ci.org/doctrine/annotations/jobs/590806588).  I'll squash the commits on my end.  Wasn't sure if you were going to squash merge them, or if I should squash them beforehand.

Reply:
We can do both. Personally, I think it's best if you compose the commit message because you are the one who knows exactly why and how these changes are related.

Reply:
I squashed the commits and updated the commit message